### PR TITLE
Functioning Subject Set Selection

### PR DESCRIPTION
**Pull request by wgranger**
This PR allows a user to select a subject set to annotate from the landing page. Hardcoded workflows and subject IDs were removed, and subject set info was added to the config.

**To consider before merge:** once this is merged, it will be more difficult to find a subject with previous annotations when debugging. Perhaps some sort of admin toggle should be added? Open to suggestions.

### CODE REVIEW

File: src/components/Home.jsx [hunk 1]
```diff
@@ -80,11 +86,15 @@ class Home extends React.Component {
           <img role="presentation" className="divider" src={Divider} />
           <div className="home-page__body-text">
             <b className="body-copy-first-word">Welcome.</b>{' '}
-            {this.props.project.description}
+            {this.props.project && this.props.project.description}
```
Comment (wgranger) [inline src/components/Home.jsx]: Existential checks were also added in a couple places in case the project has not yet been retrieved.

Comment (shaunanoordin) [reply src/components/Home.jsx]: 👍

File: src/config.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ const baseConfig = {
       host: 'https://master.pfe-preview.zooniverse.org/',
       projectId: '1764',
       projectSlug: 'wgranger-test/anti-slavery-testing',
+      workflowId: '3017'
```
Comment (shaunanoordin) [inline src/config.js]: PR #103 also adds in [Workflow ID](https://github.com/zooniverse/liberating-the-liberator/pull/103/files#diff-9d9a6cd82f41984872a66a3ab0d440c4R31) so a note has to be made during a PR rebase (depending if either 105 or 103 is merged first).

**Comment by shaunanoordin** [comment]
> To consider before merge: once this is merged, it will be more difficult to find a subject with previous annotations when debugging. Perhaps some sort of admin toggle should be added? Open to suggestions.

Question: why would it be harder to find Subjects that have had aggregated results? On localhost, isn't the default Workflow ID still 3017, and the Subjects there have aggregated results?

**Comment by wgranger** [comment]
Yes, the default workflow ID is still 3017. However, the test image we were using (with previous annotations) was from a subject set that isn't included in those year groups. In order for subject set selection to work, each subject set has to be activated on the 3017 workflow. With all subject sets activate, you're at the luck of the draw on subject sets when you choose "Transcribe Random," and the other subject sets I created for purposes of testing this branch only have one image which has no classifications attached to it. Thus, no previous annotations will be returned until classifications are made on those images.

Hope that makes sense.

**Comment by shaunanoordin** [comment]
👍  Alright, thanks to @wgranger re-enabling some Panoptes settings, I can confirm that the Subject Set selection works.

> Slack conversation, wgranger
> Subject set selection only works if the subject sets are attached to the workflow. I just disabled them for that workflow when testing something. If I turn them back on right now for 3017, you should see a difference.

This PR is now ready to merge. 👍

**Pull request merged**